src/main/webapp/script.js

```diff
@@ -78,7 +78,7 @@ function initMap() {
       window.alert("Please enter a valid location!");
       return;
     }
-
+    resizeNavButton(keepCollapsed=true);
```

Comment:
im not familiar with the javascript syntax. can you explain what does (keepCollapsed=true) do? thanks!

Reply:
This is just a named argument/parameter.

The argument keepCollapsed is used in this implementation to make sure that the filter segment stays collapsed. (Instead of the usual behaviour of toggling between collapsed/expanded)

In this case, it is effective when the filter segment is collapsed and the submit button is pressed, we want it to stay collapsed instead of toggling it and having it expanded